src/mumble/AudioInput.h

```diff
@@ -198,6 +199,7 @@ class AudioInput : public QThread {
 	int encodeOpusFrame(short *source, int size, EncodingOutputBuffer &buffer);
 
 	QElapsedTimer qetLastMuteCue;
+	AudioOutputSample *m_audioCueActive;
```

Comment:
```suggestion
	AudioOutputSample *m_activeAudioCue;
```
Otherwise, this variable sounds more like a boolean flag